service/src/main/java/org/mskcc/cmo/metadb/MetadbApp.java

```diff
@@ -28,7 +34,7 @@ public void run(String... args) throws Exception {
 
             // maybe create subscriber interface and pass handlers to messageGateway.subscribe?
             messagingGateway.subscribe(NEW_SAMPLE_INTAKE, SampleMetadataEntity.class,
-                    new SampleIntakeMessageConsumer(messagingGateway));
+                    new SampleIntakeMessageConsumer(messagingGateway, sampleMetadataRepository));
```

Comment:
Unless there is another "consumer" in this class that requires a sampleMetadataRepository, why not autowire it directly in SampleIntakeMessageConsumer?

Reply:
Tried autowiring the components to the consumer directly but we were running into issues where the components were null. 

This is temporary for now until we resolve the autowiring issue.